src/routes/get-files.ts

```diff
@@ -0,0 +1,227 @@
+import {
+  asBoolean,
+  asMap,
+  asNumber,
+  asObject,
+  asOptional,
+  asString
+} from 'cleaners'
+import Router from 'express-promise-router'
+
+import config from '../../config.json'
+import { dataStore } from '../db'
+import {
+  ApiResponse,
+  asStoreDirectory,
+  asStoreDirectoryDocument,
+  asStoreFile,
+  asStoreFileDocument,
+  asStoreRepoDocument,
+  StoreDirectoryDocument,
+  StoreFile,
+  StoreFileDocument,
+  StoreRepoDocument
+} from '../types'
+import {
+  getNameFromPath,
+  getParentPathsOfPath,
+  makeApiClientError
+} from '../utils'
+
+const { MAX_PAGE_SIZE } = config
+
+type FilesGetBody = ReturnType<typeof asFilesGetBody>
+const asFilesGetBody = asObject({
+  repoId: asString,
+  ignoreTimestamps: asOptional(asBoolean),
+  paths: asMap(asNumber)
+})
+
+interface FilesGetResponseData {
+  total: number
+  paths: FilesGetResponsePaths
+}
+
+interface FilesGetResponsePaths {
+  [path: string]: FilesGetResponseFile | FilesGetResponseDirectory
+}
+
+interface FilesGetResponseFile extends StoreFile {
+  timestamp: number
+}
+
+interface FilesGetResponseDirectory {
+  files: {
+    [path: string]: number
+  }
+  timestamp: number
+}
+
+export const getFilesRouter = Router()
+
+getFilesRouter.post('/get-files', async (req, res) => {
```

Comment:
Per coding convention, use camel case for the route names

Reply:
I'll change it to align with identifier coding conventions.